cl/recap/tasks.py

```diff
@@ -558,13 +559,23 @@ async def process_recap_docket(pk):
         await mark_pq_status(pq, msg, PROCESSING_STATUS.FAILED)
         return None
 
-    if process.current_process().daemon:
-        data = parse_docket_text(map_cl_to_pacer_id(pq.court_id), text)
-    else:
-        with concurrent.futures.ProcessPoolExecutor() as pool:
-            data = await asyncio.get_running_loop().run_in_executor(
-                pool, parse_docket_text, map_cl_to_pacer_id(pq.court_id), text
-            )
+    try:
+        if process.current_process().daemon:
+            data = parse_docket_text(map_cl_to_pacer_id(pq.court_id), text)
+        else:
+            with concurrent.futures.ProcessPoolExecutor() as pool:
+                data = await asyncio.get_running_loop().run_in_executor(
+                    pool,
+                    parse_docket_text,
+                    map_cl_to_pacer_id(pq.court_id),
+                    text,
+                )
+    except Exception as e:
+        logging.exception(e)
+        await mark_pq_status(
+            pq, traceback.format_exc(), PROCESSING_STATUS.FAILED
+        )
```

Comment:
The error message in the processing queue is currently too verbose. I propose a more concise approach: 

```suggestion
       await mark_pq_status(
            pq,
            f"We encountered a parsing error while processing this item: {e}",
            PROCESSING_STATUS.FAILED,
        )
```

Reply:
applied as suggested